Testing/WinMLRunnerTest/WinMLRunnerTest.cpp

```diff
@@ -865,7 +865,7 @@ namespace WinMLRunnerTest
         TEST_METHOD(LoadModelFailModelNotFound)
         {
             const std::wstring command = BuildCommand({ EXE_PATH, L"-model", L"invalid_model_name" });
-            Assert::AreEqual(HRESULT_FROM_WIN32(E_FAIL), RunProc((wchar_t *)command.c_str()));
+            Assert::AreEqual(HRESULT_FROM_WIN32(ERROR_FILE_NOT_FOUND), RunProc((wchar_t *)command.c_str()));
```

Comment:
Why is the cast needed?

Reply:
.c_str() produces const wchar_t* and RunProc takes in LPWSTR which is needed for CreateProcessW : https://docs.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-createprocessw

Reply:
> (wchar_t *)command.c_str()

> .c_str() produces const wchar_t*

Aren't you casting wchar_t* to wchar_t*, then?

Reply:
Oh, const cast, got it... though it's better to use const_cast

Reply:
Oh, didn't know about const_cast. Thanks, let me change it.